tests/functional/spec/features/placeholder/placeholder_default_value.spec.js

```diff
@@ -0,0 +1,18 @@
+import EmployeeNumberBlockPage from "../../../generated_pages/placeholder_default_value/employee-number-block.page";
+
+describe("Placeholder default value check", () => {
+  beforeEach("Load the survey", () => {
+    browser.openQuestionnaire("test_placeholder_default_value.json");
+  });
+
+  it("Given question regarding number of employee, when I do not enter any number and click submit, then the interstitial page shows default employee count value as 0", () => {
```

Comment:
- Can we capitalise the Given/When/Then?.
- I think it should be `Given a question with a default answer`, as that is the important bit.